CHANGELOG.rst

```diff
@@ -14,6 +14,15 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[0.14.0] - 2017-01-18
```

Comment:
I have updated PR https://github.com/edx/edx-platform/pull/14340 according to your suggestion.